### ISSUE TITLE
Warning Message: Script \life_server\init.sqf not found

### ISSUE THREAD

**Issue by Locknlol** [TAWTonic/Altis-Life#434]
Hey there! I just got back into Arma 3 and have googled quite a bit on this issue however I can't specifically locate the issue here.

I have taken "life_server" and compiled it into a .pbo, as well as moved the entire folder to the following location - @life_server/addons/life_server.pbo

I still receive this error 19:21:03 Warning Message: Script \life_server\init.sqf not found in the server console.

**Comment by danielstuart14** [comment]
Did you enable the mod in startup parameters?

**Comment by Locknlol** [comment]
This issue has been resolved by using the "4.4 Release (Stable) pre-compiled .pbo's) However, I'm still currently having issues with "extDB2" constantly giving me:

22:38:52 "extDB2: Error with Database Connection"

**Comment by Locknlol** [comment]
and since you're going to ask for it:

`[ArmaLife]
Type = MySQL
Name = AltisLife

Username = AltisLife
Password = randompasswordgen

IP = 127.0.0.1
Port = 3306
`

**Comment by Jawshy** [comment]
@Locknlol

Do you have a file called [AltisLife.sql](https://github.com/ArmaLife/Altis/blob/master/AltisLife.sql) in your Arma 3 server's directory? 

Is the database name `AltisLife`? 

For
```
CREATE DEFINER=`arma3`@`localhost`
```
Does `arma3` match a user in MySQL and `localhost` match the arma3server IP?

**Comment by Locknlol** [comment]
Yes AltisLife.sql, and I imported it into the database "AltisLife" but no, the user is/was not arma3 it was "AltisLife" as I put in my extdb-conf.ini.

I am now changing the user over to arma3 and will report back.

**Comment by Locknlol** [comment]
Database name: AltisLife
Username: arma3
Password: password1
IP Address: 127.0.0.1

Results: "extDB2: Error with Database Connection"

**Comment by Jawshy** [comment]
Is there a user called `arma3` or `AltisLife` in your phpMyAdmin/MySQL and does that user have the necessary privileges (all privileges)?

**Comment by Locknlol** [comment]
Yes to both. I have done it both ways.

**Comment by Jawshy** [comment]
I'm afraid that's all I can do to help. @danielstuart14 may be able to help you further, but he won't be on again for a number of hours.

**Comment by Locknlol** [comment]
New issue...

```
[00:54:50 --5:00] [Thread 5043] extDB2: Database Type: MySQL
[00:54:50 --5:00] [Thread 5043] extDB2: Database Exception Error: Connection attempt failed: Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2)
 0:54:50 "extDB2: Error with Database Connection"
```

**Comment by ColinM9991** [comment]
These are not issues with the Altis Life repository.

**Comment by torchgodz** [comment]
How are you using extdb2? Are you using it as a mod "@extDB2" or just having extDB2 in the root folder and the extDB2.ini and extDB.dll also in the root folder?  Also have you unblocked the extDB2.dll file.

**Comment by Locknlol** [comment]
Linux, so no need to unblock a .dll, additionally it's in "@extDB2"

**Comment by torchgodz** [comment]
Are you running it as a mod or server side mod mod=@extDB2, or server mod=@extDB2?

**Comment by Locknlol** [comment]
Mod mod on both extdb2 and life_server

**Comment by torchgodz** [comment]
Ok then one more question are the extDB2.dll and extDB2.ini in the @extDB2 file or where are they?

**Comment by Locknlol** [comment]
Issue resolved:

ORIGINAL:
```
[ArmaLife]
Type = MySQL
Name = AltisLife

Username = AltisLife
Password = randompassword

IP = localhost
Port = 3306
```

CHANGED TO:

```
[ArmaLife]
Type = MySQL
Name = AltisLife

Username = AltisLife
Password = randompassword

IP = 127.0.0.1
Port = 3306
```

MySQL - "localhost" means "use the mysql.sock" - where "127.0.0.1" means "use TCP/IP Socket"

**Comment by torchgodz** [comment]
Good glad to help. Sometimes just gotta ask the basic questions.

**Comment by Locknlol** [comment]
How do I mark as solved?
